packages/lucide-vue/src/createVueComponent.ts

```diff
@@ -47,6 +49,13 @@ export default (iconName: string, iconNode: IconNode): Component => ({
       children = [],
     },
   ) {
+    if (showDeprecationWarning) {
+      console.warn(
+        '[Lucide Vue] This package will be deprecated end of 2023. Please use upgrade to Vue 3 and use the latest lucide package for Vue.',
```

Comment:
`Please upgrade to Vue 3 and use the latest lucide package for Vue.'`

Reply:
Thanks!